src/main/java/com/tabnine/UninstallListener.kt

```diff
@@ -10,13 +10,19 @@ import com.tabnine.general.StaticConfig
 import com.tabnine.general.readTempTabninePluginZip
 import com.tabnine.lifecycle.UninstallReporter
 import java.nio.file.Paths
+import java.time.Duration
 
 private const val TABNINE_JAR_NAME = "TabNine-"
 private const val JAR_SUFFIX = ".jar"
-private val TABNINE_JAR_REGEX = """^$TABNINE_JAR_NAME\d+\.\d+\.\d+.jar""".toRegex()
+private val TABNINE_JAR_REGEX = """^$TABNINE_JAR_NAME\d+\.\d+\.\d+(-\w+.\d+)*.jar""".toRegex()
```

Comment:
catches `-<something>.<number>` 0 or more times, i can make it accept only 0 or 1 times if you think its necessary @amircodota